packages/grafana-ui/src/themes/_variables.scss.tmpl.ts

```diff
@@ -193,6 +193,7 @@ $btn-semi-transparent: rgba(0, 0, 0, 0.2) !default;
 
 // sidemenu
 $side-menu-width: 60px;
+$navbar-padding: 20px;
```

Comment:
I made the change in this file and the `public/sass/_variables.generated.scss` got updated, but not sure this it the correct way to add variables. @dprokop

Reply:
Yes, that's the correct way. Ideally you should use some of the variables specified in TS theme (`grafana-ui/src/themes/default.ts`) but i see there is not spacing variable defined for 20px.